### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by shubham1091**
there were some links were missing and the blockchain was 2 times so i updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,8 +7,8 @@ This repository contains the list of communities and job portals you can join an
 ## Roadmaps:
 <p align="center"><img align="center" alt="Coding" src="https://media.giphy.com/media/xUySTOigOUHucl3rfW/giphy.gif"></p>
 <p  align="center"><a href="https://github.com/commclassroom/roadmaps/tree/main/Backend-Development#readme"><B>Backend-Development</B></a></p>
-<p  align="center"><a href=""><b>Blockchain</b></a></p>
-<p  align="center"><a href=""><b>DevOps</b></a></p>
+<!-- <p  align="center"><a href=""><b>Blockchain</b></a></p> -->
```
Comment (kaiwalyakoparkar) [inline README.md:10]: Please remove this comment if you are making changes to it.

File: README.md [hunk 2]
```diff
@@ -7,8 +7,8 @@ This repository contains the list of communities and job portals you can join an
...
+<p  align="center"><a href="https://github.com/commclassroom/roadmaps/blob/main/DevOps/Readme.md"><b>DevOps</b></a></p>
```
Comment (kaiwalyakoparkar) [inline README.md:11]: Please use relative linking and not the entire url. Please take a look at this [stack overflow website](https://stackoverflow.com/questions/7653483/github-relative-link-in-markdown-file)

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

**Comment by shubham1091** [comment]
Done👍

**Comment by shubham1091** [comment]
Hi do you want me to change anything else in this pr

File: README.md [hunk 3]
```diff
@@ -5,22 +5,24 @@ Are you prepared to embark on your quest to receive mentoring and land a job? Yo
...
-<p  align="center"><a href="https://github.com/commclassroom/roadmaps/tree/main/DevRel#readme"><b>DevRel</b></a></p>
-<p  align="center"><a href="https://github.com/commclassroom/roadmaps/tree/main/Frontend-Development#readme"><b>Frontend-Development</b></a></p>
-<p  align="center"><a href="https://github.com/commclassroom/roadmaps/tree/main/Fullstack-Development#readme"><b>Fullstack-Development</b></a></p>
-<p  align="center"><a href=""><b>Android-Development</b></a></p>
-<p align="center"><a href="https://github.com/commclassroom/roadmaps/tree/main/Open-Source#readme"><b>Blockchain</b></a></p>
+<p  align="center"><a href="/Backend-Development/README.md"><B>Backend-Development</B></a></p>
+<p  align="center"><a href="/Blockchain/readme.md"><b>Blockchain</b></a></p>
+<p  align="center"><a href="/DevOps/Readme.md"><b>DevOps</b></a></p>
+<p  align="center"><a href="/DevRel/Readme.md"><b>DevRel</b></a></p>
+<p  align="center"><a href="/Frontend-DEVlopment/README.md"><b>Frontend-Development</b></a></p>
```
Comment (siddhant-khisty) [inline README.md:14]: the frontend roadmap has a typo which is causing a 404. Please fix it.

**Review by siddhant-khisty — CHANGES_REQUESTED** [review]

**Comment by siddhant-khisty** [comment]
Closing due to inactivity. And the issue seems to be fixed by a different PR

**Pull request closed without merging**